main.js

```diff
@@ -1,112 +1,41 @@
-var mealsJSON = {
-  'meals': {
-    'breakfast':[
-      {
-        'name': 'Breakfast Burrito',
-        'ingredients': {'tortilla': 1, 'eggs': 2, 'potatoes': 1, 'cheese (in cups)': 0.125, 'onion': 0.5}
-      },
-      {
-        'name': 'Avocado Toast',
-        'ingredients': {'bread': 2, 'avocado': 1, 'hummus': 0.25}
-      },
-      {
-        'name': 'Yogurt Parfait',
-        'ingredients': ['yogurt', 'berries', 'granola']
-      },
-      {
-        'name': 'Fried Potatoes',
-        'ingredients': ['potatoes', 'onion', 'coconut oil', 'avocado', 'cheese']
-      }
-    ],
-    'lunch': [
-      {
-        'name': 'Veggie BLT Sandwich',
-        'ingredients': ['bread', 'facon', 'avocado', 'lettuce', 'tomato', 'mayo']
-      },
-      {
-        'name': 'Veggie Burrito',
-        'ingredients': ['tortilla', 'rice', 'beans', 'cilantro', 'salsa', 'avocado', 'yellow squash', 'lettuce', 'sour cream', 'cheese']
-      },
-      {
-        'name': 'Falafel Wrap',
-        'ingredients': ['tortilla', 'hummus', 'falafel', 'tabboleh', 'onion', 'raddish']
-      },
-      {
-        'name': 'Pasta Salad',
-        'ingredients': ['spiral noodles', 'ranch', 'mayo', 'parsley', 'bell peppers', 'onion']
-      }
-    ],
-    'dinner': [
-      {
-        'name': 'Sweet & Sour Tofu',
-        'ingredients': ['tofu', 'pineapple', 'onion', 'bell peppers', 'soy sauce', 'ketchup', 'rice vinegar', 'corn starch', 'brown sugar', 'garlic', 'ginger']
-      },
-      {
-        'name': 'Spaghetti',
-        'ingredients': ['spaghetti noodles', 'marinara', 'olive oil', 'garlic', 'lettuce', 'cucumber', 'cherry tomatoes', 'ranch']
-      },
-      {
-        'name': 'Self-Made TV Dinner >_<',
-        'ingredients': ['soy chicken nugz', 'potatoes', 'butter', 'corn', 'soymilk', 'ketchup', 'baked beans', 'broccoli']
-      }
-    ]
-  }
-
-};
-if(!localStorage.mealObj) {
-  localStorage.setItem('mealObj', JSON.stringify(mealsJSON));
-}
+// I want to make meals into instances of a class called meal, I also want to change the layout of a meal to have
+// a name, ingredients with straightforward quantities, a sauce & topping & spices key with all the harder to measure out ingredients as the value.
+// Finally, for the list I will need to loop through the ingreidents in every meal and add up the quantities to display them. Then I will need to do something
+// similar for the topppings section.
+
+
+
+
+
+//pretend mealsJSON came from localcstorange instead
+var mealList = new MealList();
+
+mealList.load();
+
+
 var breakfast = $('#saved-meals-breakfast');
 var lunch = $('#saved-meals-lunch');
 var dinner = $('#saved-meals-dinner');
-var mealsParsed = JSON.parse(localStorage.mealObj).meals
```

Comment:
can localStorage be used like this or does it need to be "localStorage.getItem(...)"?